frontend/src/layout/navigation/TopBar/notificationsLogic.ts

```diff
@@ -19,11 +19,15 @@ export const notificationsLogic = kea<notificationsLogicType>([
         setMarkReadTimeout: (markReadTimeout: number) => ({ markReadTimeout }),
         incrementErrorCount: true,
         clearErrorCount: true,
+        markAllAsRead: true,
     }),
     loaders(({ actions, values }) => ({
         importantChanges: [
             [] as HumanizedActivityLogItem[],
             {
+                markAllAsRead: () => {
+                    return values.importantChanges.map((ic) => ({ ...ic, unread: false }))
```

Comment:
I'm only now seeing notifications code and a small nit from my side is us using the negative here i.e. `unread` rather than `read`

Reply:
Yes! Stood out for me coming back to this code today...

The power of hackathon code hitting prod 🤣